.cirrus.yml

```diff
@@ -8,10 +8,10 @@ docker_builder:
     # from https://flutter.dev/docs/development/tools/sdk/releases
     matrix:
     - DOCKER_TAG: latest
-      FLUTTER_VERSION: 3.3.4
+      FLUTTER_VERSION: 3.4.0-34.1.pre
```

Comment:
latest defaults to the latest stable. could you please revert this line?

Reply:
Done